examples/lusid/transactions/upload-transactions-from-csv.sql

```diff
@@ -0,0 +1,43 @@
+-- ============================================================
+
+-- Description:
+-- 1. In this query, we run an ETL process on some transactions.
+-- 2. First, we load a CSV file of transactions from Drive.
+-- 3. Next, we transform the shape of the transaction data.
+-- 4. Finally we upload the transaction data into LUSID.
+
+-- ============================================================
+
+-- Extract transaction data from LUSID Drive
+@txn_data = 
+use Drive.Csv
+--file=/luminesce-examples/transactions.csv
+enduse;
+
+-- Set variables for the portfolio's scope and code
+@@portfolio_scope =
+select 'IBOR';
```

Comment:
can we put this all on one line?
Same for other files.

Reply:
Changes made.